packages/gravity-ui-web/src/sass/03-elements/_headings.scss

```diff
@@ -4,11 +4,30 @@ h3,
 h4,
 h5,
 h6 {
-  @include gravy-font(secondary, regular);
+  font-weight: grav-font-weight('primary', 'bold');
+}
+
+h1 {
+  @include grav-font-size(6);
+}
+
+h2 {
+  @include grav-font-size(5);
+}
+
+
```

Comment:
Might want to include a rule to remove redundant whitespace.

Reply:
Yup. I've manually fixed this like a savage for now though.